TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -219,6 +219,79 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
+    },
+    quarkus_startup: {
+        outerRegex: /Quarkus Test:  measure \d*? ([\s\S\n]*)/,
+        metrics: {
+            "Footprint": {
+                //Example: Footprint for  warmup 0 10129332
+                regex:/Footprint for  measure \d*? (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "kb",
+                },
+            "Startup time in ms": {
+                //Example: Startup time for measure 0 : 53232
+                regex:/Startup time for measure \d*? : (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "ms",   
+            }
+        }
+    },
+    pingperf_baremetal: {
+        outerRegex:/Running \d*? measures([\s\S\n]*)/,
+        metrics: {
+    /*        "Container_Footprint": {
+                //Example: Footprint for  warmup 0 10129332
+                regex:/Footprint for  measure \d*? (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "kb",
+                },
+            "Container_Startup time in ms": {
+                //Example: Startup time for measure 0 : 53232
+                regex:/Startup time for measure \d*? : (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "ms",   
+            },
+    */
```

Comment:
Please do not check in the code that is commented out.

Reply:
sorry removed the comment